newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -149,9 +149,10 @@ const ExtensionInstallDialog = ({
             ]
           : undefined
       }
-      cannotBeDismissed={false}
       open
+      cannotBeDismissed={isInstalling}
       onRequestClose={onClose}
+      onApply={onInstallExtension}
```

Comment:
Should dismiss action still install the extension (even with save on dismiss enabled)?

Reply:
Good question. 
Again, I think that if you have configured your IDE to apply on click outside... (which is not the default behavior) then we should do it here too, even if it's a bit weird?
plus, if you misclicked, then you can uninstall it really quickly

Reply:
Will this overwrite an extension that has changes, or will there at least be a warning?

Reply:
![image](https://user-images.githubusercontent.com/4895034/171866469-b76ec157-8515-4fd9-99d5-1383d3d2baf1.png)

There is a warning in place yes!